### DISCUSSION TITLE
Cannot find my achievements anymore

### DISCUSSION THREAD

**Discussion by AyaMohamed** [community/community#40658] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I hid my "Pull Shark" achievement by mistake, and i cannot find it on my profile anymore

**Comment by radar-blip** [comment]
Check here for it https://github.com/AyaMohamed?tab=achievements
if its there, unhide it. 😃

**Comment by jfullstackdev** [comment]
seems it's fixed by now 

![image](https://user-images.githubusercontent.com/47092464/205429714-e06f90c3-f354-48d2-aebb-fe48133be0e0.png)

**Comment by AyaMohamed** [comment, marked as answer]
Yes it's fixed.
Thanks
